app/views/student_quiz_mailer/quiz_email.html.slim

```diff
@@ -2,9 +2,12 @@ html lang="en"
 head
   meta content="text/html; charset=UTF-8" http-equiv="Content-Type"
 body
-  p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse rhoncus iaculis orci, non accumsan nulla eleifend in. Duis sed quam sed felis feugiat efficitur. Praesent rutrum nibh orci, a commodo lacus condimentum nec.
+  p
+    | Congratulations on completing the #{@quiz.name.sub(' Quiz', '')}. Before your class moves on
```

Comment:
This seems like something that a Decorator can handle as well. (See my comments below for what that is.)